src/main/java/seedu/contax/logic/parser/ParserUtil.java

```diff
@@ -132,6 +134,40 @@ public static Tag parseTag(String tag) throws ParseException {
         return tagSet;
     }
 
+    /**
+     * Parses a {@code String filePath} into a {@code Path}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code phone} is invalid.
```

Comment:
The javadoc here is probably wrong.
```suggestion
     * @throws ParseException if the given {@code filePath} is invalid.
```

Reply:
thanks for catching!